tests/classmethods_test.py

```diff
@@ -105,3 +106,107 @@ def testPreservesClassArgumentAfterUnstub(self):
         self.assertEqual("Cat foo", Cat.meow("foo"))
 
 
+class Retriever:
+    @classmethod
+    def retrieve(cls, item):
+        return item
+
+
+class TrickDog(Dog, Retriever):
+    pass
+
+
+class InheritedClassMethodsTest(TestBase):
+
+    def tearDown(self):
+        unstub()
+
+    def testUnstubs(self):
+        when(TrickDog).bark().thenReturn("miau!")
+        when(TrickDog).retrieve("stick").thenReturn("ball")
+        unstub()
+        self.assertEqual("woof!", TrickDog.bark())
+        self.assertEqual("stick", TrickDog.retrieve("stick"))
+
+    # TODO decent test case please :) without testing irrelevant implementation
+    # details
+    #
+    # TODO This test does not work for inherited @classmethods. but that does
```

Comment:
That comment is based on the following finding in the Python interpreter: 

```
>>> class Dog():
...     @classmethod
...     def bark(cls):
...         return "woof!"
... 
>>> class Retriever:
...     @classmethod
...     def retrieve(cls, item):
...         return item
... 
>>> 
>>> class TrickDog(Dog, Retriever):
...     pass
... 
>>> isinstance(Dog.__dict__.get("bark"), classmethod)
True
>>> isinstance(TrickDog.__dict__.get("bark"), classmethod)
False
>>> 
```

Reply:
Hm, yeah sure, the `dict` of `TrickDog` is basically empty and `None`is not a `classmethod`.